src/components/navigation/navigation-disclosure-panel.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Disclosure } from "@headlessui/react";
+import { classNames } from "../../util/class-names";
+
+export interface NavigationDisclosurePanelItemProps {
+    children: React.ReactNode;
+    isActive?: boolean;
+    isIndented?: boolean;
+}
+
+const NavigationDisclosurePanelItem = ({
+    children,
+    isActive,
+    isIndented,
+}: NavigationDisclosurePanelItemProps) => {
+    return (
+        <div
+            className={classNames(
+                "relative w-full px-8 py-3 text-left text-sm text-neutral-0 hover:bg-primary-900+10",
+                isIndented && "px-14",
```

Comment:
Is it right to have a padding on the left and right side? I mean, for now it really doesn’t matter, but we could add some more space for any future nav item which text could be get longer? Just a thought … nothing wyyyyld